routes/item.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/view', (req, res, next) => {
```

Comment:
all of the router functions have the same problem- pls just add in ); at the end of the function like so:
`router.get('/view', (req, res, next) => {
});`